tags/views.py

```diff
@@ -785,14 +979,23 @@ def page_summary(request):
     'data': data,
   }
 
+'''
+load comments for each highlight using value tags
+'''
 @login_required
 @ajax_request
-def comments_by_highlight(request):
+def comments_by_highlight_tags(request):
```

Comment:
ditto, looks like this wasn't updated in urls.py

Reply:
yes sorry. All those need to be in urls. Slipped my mind